projects/u-mill/white_mesa/decantation_filtration.py

```diff
@@ -11,9 +11,9 @@
              |   DECANTATION   |
  STD         |                 |
     <--------|  + Single Tank  |<-------- Wash water (internal)
- underflow   |                 |<-------- Pre-Leach Feed (from Leaching Module pre-leach product)
```

Comment:
Why are you removing this?

Reply:
This must have reverted back by accident. I had to redownload the entire repository due to the extent of conflicts in my version of the code. This can be put back.